Networking/Networking/Remote/WCPayRemote.swift

```diff
@@ -19,4 +18,37 @@ public class WCPayRemote: Remote {
 
         enqueue(request, mapper: mapper, completion: completion)
     }
+
+    /// Loads a WCPay account for a given site ID and parses the rsponse
+    /// - Parameters:
+    ///   - siteID: Site for which we'll fetch the WCPay account info.
+    ///   - completion: Closure to be executed upon completion.
+    public func loadAccount(for siteID: Int64,
+                            completion: @escaping (Result<WCPayAccount, Error>) -> Void) {
+        let path = "payments/accounts"
+
+        let parameters = [AccountParameterKeys.fields: AccountParameterValues.fieldValues]
+
+        let request = JetpackRequest(wooApiVersion: .mark3, method: .get, siteID: siteID, path: path, parameters: parameters)
+
+        let mapper = WCPayAccountMapper()
+
+        enqueue(request, mapper: mapper, completion: completion)
+    }
+}
+
+// MARK: - Constants!
+//
+private extension WCPayRemote {
+
+    enum AccountParameterKeys {
+        static let fields: String = "_fields"
+    }
+
+    enum AccountParameterValues {
+        static let fieldValues: String = """
+            status,has_pending_requirements,has_overdue_requirements,current_deadline,
+            statement_descriptor,store_currencies,country
+            """
```

Comment:
🛑  We would most probably want to remove the line break here. This is what gets encoded in the `parameters` dictionary in `loadAccount()`:

```
(lldb) po parameters
▿ 1 element
  ▿ 0 : 2 elements
    - key : "_fields"
    - value : "status,has_pending_requirements,has_overdue_requirements,current_deadline,\nstatement_descriptor,store_currencies,country"
```

Notice the `\n`. 😬 

We can remove the line break by adding a backslash:

```suggestion
        static let fieldValues = """
            status,has_pending_requirements,has_overdue_requirements,current_deadline,\
            statement_descriptor,store_currencies,country
            """
```

Reply:
Eeek! Thank you for catching that! Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3881/commits/eda4b4fd272f3b0a4f03023af1d8f934e5e78eca